src/library-authoring/LibraryAuthoringPage.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect, useState } from 'react';
+import { StudioFooter } from '@edx/frontend-component-footer';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Container, Icon, IconButton, SearchField, Tab, Tabs,
+} from '@openedx/paragon';
+import { InfoOutline } from '@openedx/paragon/icons';
+import {
+  Routes, Route, useLocation, useNavigate, useParams,
+} from 'react-router-dom';
+
+import Loading from '../generic/Loading';
+import SubHeader from '../generic/sub-header/SubHeader';
+import Header from '../header';
+import NotFoundAlert from '../generic/NotFoundAlert';
+import LibraryComponents from './LibraryComponents';
+import LibraryCollections from './LibraryCollections';
+import LibraryHome from './LibraryHome';
+import { useContentLibrary } from './data/apiHook';
+import messages from './messages';
+
+const TAB_LIST = {
+  home: '',
+  components: 'components',
+  collections: 'collections',
+};
+
```

Comment:
Since we're using TypeScript now, this could be an enum?

Reply:
Great! Done: 3b2a6ada12210ba15a716d3e302884565dc4b998